### ISSUE TITLE
ObjectRocket connection string

### ISSUE THREAD

**Issue by Evanion** [balderdashy/sails-mongo#179]
I'm trying to use Object Rockets SAAS. But I can't seem to get the configuration correct.
The connection string supplied by ObjectRocket: 

````
< replicaset >/< host 1 >:< port >,< host 2 >:< port >/< dbName >
````

my config:

````
stageMongodbServer: {
    adapter: 'sails-mongo',
    url: 'mongodb://< replicaset >/< host 1 >:< port >,< host 2 >:< port >/< dbName >'
}
````

But sails gives me the following error:

````
if(databaseName.indexOf(invalidChars[i]) != -1) throw new Error("database 
                                                          ^
Error: database names cannot contain the character '.'
````

I have also tested the following:

`````
adapter: 'sails-mongo',
                host: 'lon-c9-0.objectrocket.com',
                port: 43088,
                user: '< user >',
                password: '< password >',
                database: '< dbname >',
                replSet: {
                        servers: [
                                {
                                        host: 'lon-c9-0.objectrocket.com',
                                        port: 43088
                                },
                                {
                                        host: 'lon-c9-1.objectrocket.com',
                                        port: 43088
                                }
                        ],
                        options: {
                                rs_name: '< replica set name >'
                        }
                }
`````

And it still complains that the db name contains a '.' even though it only contains letters and '_'

**Comment by sailsbot** [comment]
Thanks for posting, @Evanion.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
